local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Should this be checked in?

Reply:
Really sorry normally it is added to the git ignore file, will remove and commit